act-rules-format.bs

```diff
@@ -315,31 +315,28 @@ The definition of a rule MUST always result in one of the following outcomes:
 </div>
 
 
-Rule Quality Assurance {#quality}
-=================================
+Test Cases {#test-cases}
```

Comment:
Should this be linked from the [ACT Rule Structure](https://pr-preview.s3.amazonaws.com/w3c/wcag-act/pull/276.html#act-rule-structure)? I raised an issue on this and related questions last week: https://github.com/w3c/wcag-act/issues/272